### PR TITLE
fix #791, clarify when FocusRings are enabled in docs

### LINKED ISSUE #791: Focus Ring enables when switching tabs with mouse
# 🐛 Bug Report 

When checking out your [Radio Group](https://react-spectrum.adobe.com/react-spectrum/RadioGroup.html), I noticed the blue Focus Ring is enabled when I:

1. Select radio item
2. Use mouse to click on another browser tab.
3. Use mouse to click on the Radio Group tab to return to the page
4. The Focus Ring is shown, when it was not shown at initial selection.

## 💁 Possible Solutions

* Can be helpful to know what 'last selection' was when navigating in the middle of filling out forms for those with attention disorders (like me ;) ).
  * Should be [documented](https://github.com/adobe/react-spectrum/blob/main/packages/@react-aria/focus/docs/FocusRing.mdx#introduction).
* Want to be consistent
  * Needs automated tab-switching test, maybe visual-diff?
  * Extra code in RocusRing to ensure a check for tab-swtich is done before enabling.

I'm indifferent, but I lean towards the first option which is an increase of usability IMHO.

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| react-spectrum   | 3.0.2
| Browser          | Firefox & Vivaldi (Chromium)
-- | --
| Operating System | Win10

## 🧢 Your Company/Team

Web users

### PR DESCRIPTION
**Pull request by tomByrer**
Closes #791 
## ✅ Pull Request Checklist:
- [x] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).

### CODE REVIEW

File: packages/@react-aria/focus/docs/FocusRing.mdx [hunk 1]
```diff
@@ -35,8 +35,10 @@ keywords: [focus, keyboard focus, focus management, aria]
 
 `FocusRing` is a utility component that can be used to apply a CSS class when an element has keyboard focus.
 This helps keyboard users determine which element on a page or in an application has keyboard focus as they
-navigate around. Focus rings are only visible when interacting with a keyboard so as not to distract mouse
-and touch screen users.
+navigate around. Focus rings are usually suppresed for mouse and touch screen users, unless browser tabs are
+switched away and back, or while navigating a form with the next/previous buttons on iOS.
+
+Cha
```
Comment (snowystinger) [inline packages/@react-aria/focus/docs/FocusRing.mdx:41]: Cha?

I think we want to keep the information about why the focus rings are suppressed for mouse/touch users.
Maybe we leave it as it was and add a sentence saying `In cases where we are unable to detect if the user is using a mouse or touch screen, such as switching in from a different tab, we show the focus ring.`

Comment (majornista) [reply packages/@react-aria/focus/docs/FocusRing.mdx:41]: "suppresed" is spelled incorrectly in submitted PR.

Comment (tomByrer) [reply packages/@react-aria/focus/docs/FocusRing.mdx:41]: yes, good suggestion!

I was unsure of the wording.

**Comment by tomByrer** [comment]
Closed in favor of #814

**Pull request closed without merging**